man/man1/oci-image-migrate.1

```diff
@@ -54,7 +54,7 @@ The full path name of the on-premise custom image. This option is mandatory.
 .IP "-b|--bucket"
```

Comment:
This is a another FILE comment, not for this line, the top license block need to be completed